daemon/container.go

```diff
@@ -1441,6 +1442,18 @@ func (container *Container) startLogging() error {
 			return err
 		}
 		l = dl
+	case "gelf":
+		metadata := gelf.GelfMetadata{
```

Comment:
can we create a custom struct that encapsulate as many infos as needed by a logger about a container (and have a method that "translate" a container to this struct, cause I don't like having the daemon struct in loggers :D)? So I agree with @LK4D4 that less moving parts is better. Also I've seen the other loggers need different infos (all coming from Container) to use it for log messages and tag so we can directly pass this struct that is a representation of the container instead of lots of different structured infos. @LK4D4 @mariussturm wdyt? does what I said make sense? :smile: